pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -360,26 +396,40 @@ message Capability {
     repeated string drop_capabilities = 2;
 }
 
-// LinuxContainerConfig contains platform-specific configuration for
-// Linux-based containers.
-message LinuxContainerConfig {
-    // Resources specification for the container.
-    optional LinuxContainerResources resources = 1;
+// SecurityContext holds security configuration that will be applied to a container.
+// Some fields are present in both SecurityContext and PodSecurityContext.  When both
+// are set, the values in SecurityContext take precedence.
+message SecurityContext {
     // Capabilities to add or drop.
-    optional Capability capabilities = 2;
+    optional Capability capabilities = 1;
+    // If set, run container in privileged mode.
+    // Processes in privileged containers are essentially equivalent to root on the host.
```

Comment:
It isn't clear how this is translated in case of user namespaces. So, maybe leave out this sentence?